presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/execution/SqlFunctionExecutors.java

```diff
@@ -38,10 +41,10 @@
     private final ThriftSqlFunctionExecutor thriftSqlFunctionExecutor;
 
     @Inject
-    public SqlFunctionExecutors(Map<Language, FunctionImplementationType> supportedLanguages, ThriftSqlFunctionExecutor thriftSqlFunctionExecutor)
+    public SqlFunctionExecutors(Map<Language, FunctionImplementationType> supportedLanguages, @Nullable ThriftSqlFunctionExecutor thriftSqlFunctionExecutor)
```

Comment:
Can we just make this `Optional`? :)

Reply:
I would also prefer Optional here.

Reply:
Have you not pushed yet since it's still @Nullable?

Reply:
Decided to go with nullable for injection but change the member variable to optional. This for the most part avoids potential bugs of having a nullable variable, and doesn't require me to figure out how to inject Optional 😂